zeppelin-web/pom.xml

```diff
@@ -33,9 +33,9 @@
   <name>Zeppelin: web Application</name>
 
   <!-- See https://github.com/eirslett/frontend-maven-plugin/issues/229 -->
-  <prerequisites>
-    <maven>3.1.0</maven>
-  </prerequisites>
+  <!--<prerequisites>-->
+    <!--<maven>3.1.0</maven>-->
```

Comment:
I'm not sure I understand what you mean, but I do not see a reason to remote enforcement of the minimum maven version. It was added there to fail build fast and avoid mentioned bug.